src/components/ScholarshipAmountField.jsx

```diff
@@ -79,7 +79,11 @@ function ScholarshipAmountField(props) {
             className={classes.amountSelect}
             variant="outlined"
             value={amountType}
-            onChange={formik.handleChange}>
+            onChange={(e) => {
+              formik.setFieldValue('amount.min', 0);
+              formik.setFieldValue('amount.max', 0);
+              formik.handleChange('amount.type')(e);
```

Comment:
got this from [here.](https://stackoverflow.com/questions/61384089/how-to-call-two-function-in-onchange-using-formik)